documentation/docs/developers/android-assets.md

```diff
@@ -0,0 +1,56 @@
+# Android Assets
```

Comment:
Thanks for adding this, reads very well and handy to have all the extra links and info sources too